### DISCUSSION TITLE
File.browserForOpen/browserForOpenMultiple doesn't work on Mac OS 12.1

### DISCUSSION THREAD

**Discussion by aW4KeNiNG** [airsdk/Adobe-Runtime-Support#1458] [Q&A]
A few days ago, Apple has released a new Mac OS version (12.1). With the new version I have 3 customers with the same problem on that version.

When `File.browserForOpen` or `File.browserForOpenMultiple` method is called, it works fine the first time, but the second time it is completely freezed (the OS window to select the file), and it requires to force quit the application.

The AIR version is 33.1.1.633, but I have sent a custom build with the new AIR 33.1.1.713 with the same bad result.

Do you have the same problem?

**Comment by ajwfrost** [comment]
Oddly I thought we'd already fixed this issue from #763 but it looks like there was a fix for #653 which perhaps was a similar effect...

We can check again on this one...

**Reply by aW4KeNiNG** [reply]
Yes please, let me know the results and when it could be fixed.

Thanks.

**Reply by aW4KeNiNG** [reply]
I already have 6 users with the same problem, we have updated to the latest version and I can confirm the problem myself.

**Reply by ajwfrost** [reply]
Reproduced here too... there were some fixes in there that should have solved this (well, #763) which were tagged as "AIR-4183" in our system, fixed on 21 April so should have been in 33.1.1.476 - not sure why it's missing from the release notes...! 

 .... actually we've just found the issue, it's the same problem but for some reason there is now a different code path being executed within the runtime that didn't have the fix on it. Will just get this confirmed and tidied up but we should be able to get an updated release done next week to fix this...

thanks
